checker/src/org/checkerframework/checker/fenum/qual/FenumBottom.java

```diff
@@ -1,14 +1,18 @@
 package org.checkerframework.checker.fenum.qual;
 
-import java.lang.annotation.*;
-
-import com.sun.source.tree.Tree;
-
 import org.checkerframework.framework.qual.DefaultFor;
-import org.checkerframework.framework.qual.TypeUseLocation;
 import org.checkerframework.framework.qual.ImplicitFor;
 import org.checkerframework.framework.qual.SubtypeOf;
 import org.checkerframework.framework.qual.TargetLocations;
+import org.checkerframework.framework.qual.TypeUseLocation;
+
+import java.lang.annotation.Documented;
```

Comment:
Your diff (that we are commenting on) has java.* coming after org.checkerframework.*, but the file [checker/eclipse/checker-framework.importorder](
https://github.com/typetools/checker-framework/blob/master/checker/eclipse/checker-framework.importorder) has them in the opposite order.  So maybe your IntelliJ configuration isn't true to the Eclipse configuration file, or maybe I don't understand the Eclipse configuration file.